compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -63,6 +65,9 @@ struct ROCmOptions {
     static cl::OptionCategory category("ROCm HAL Target");
     binder.opt<std::string>("iree-rocm-target-chip", targetChip,
                             cl::cat(category), cl::desc("ROCm target chip."));
+    binder.opt<std::string>(
+        "iree-rocm-target-features", targetFeatures, cl::cat(category),
+        cl::desc("ROCm target features; e.g., '+sramecc,+xnack'."));
```

Comment:
Would we want to use cl::list here? I haven't seen it used in this context so I'm not sure if it's not supported or has some other isssues.

Reply:
Features are typically given as a comma-listed string directly. We also do this on the CPU side. So I think it's fine for now.